testing/tests/DevExpress.ui.widgets.scheduler/integration.workSpace.tests.js

```diff
@@ -912,6 +911,71 @@ QUnit.test("timeCellTemplate should have correct options", function(assert) {
     assert.equal(templateOptions.text, "3:00 AM", "text options is ok");
 });
 
+QUnit.test("timeCellTemplate should contains the date field of data parameter in the Day view", function(assert) {
+    var resultDates = [];
+    this.createInstance({
+        currentView: "day",
+        views: ["day"],
+        currentDate: new Date(2016, 8, 5),
+        startDayHour: 0,
+        endDayHour: 4,
+        cellDuration: 60,
+        timeCellTemplate: function(itemData) {
+            resultDates.push(itemData.date);
+        }
+    });
+
+    assert.equal(resultDates.length, 4);
+    assert.deepEqual(resultDates[0], new Date(2016, 8, 5));
+    assert.deepEqual(resultDates[1], new Date(2016, 8, 5, 1));
+    assert.deepEqual(resultDates[2], new Date(2016, 8, 5, 2));
+    assert.deepEqual(resultDates[3], new Date(2016, 8, 5, 3));
+});
+
+QUnit.test("timeCellTemplate should contains the date field of data parameter in Week view", function(assert) {
+    var resultDates = [];
+    this.createInstance({
+        currentView: "week",
+        views: ["week"],
+        currentDate: new Date(2016, 8, 5),
+        firstDayOfWeek: 0,
+        startDayHour: 0,
+        endDayHour: 4,
+        cellDuration: 60,
+        timeCellTemplate: function(itemData) {
+            resultDates.push(itemData.date);
+        }
+    });
+
+    assert.equal(resultDates.length, 4);
+    assert.deepEqual(resultDates[0], new Date(2016, 8, 4));
+    assert.deepEqual(resultDates[1], new Date(2016, 8, 4, 1));
+    assert.deepEqual(resultDates[2], new Date(2016, 8, 4, 2));
+    assert.deepEqual(resultDates[3], new Date(2016, 8, 4, 3));
+});
+
+QUnit.test("timeCellTemplate should contains the date field of data parameter in Week view", function(assert) {
```

Comment:
test caption: 'in WorkWeek view' instead of 'Week'

Reply:
Fixed)